service/policy/resourcemapping/resource_mapping.proto

```diff
@@ -8,6 +8,61 @@ import "google/api/annotations.proto";
 import "common/common.proto";
 import "policy/objects.proto";
 
+/*
+   Resource Mapping Groups
+*/
+
+message ListResourceMappingGroupsRequest {
+  string namespace_id = 1;
+}
+
+message ListResourceMappingGroupsResponse {
+  repeated ResourceMappingGroup resource_mapping_groups = 1;
+}
+
+message ListResourceMappingGroupsByFqnsRequest {
+  repeated string fqns = 1;
```

Comment:
```suggestion
  Structure of the RM group FQN is 'https://<namespace>/resm/<group name>'
  repeated string fqns = 1 [(buf.validate.field).required = true];
```

You can also make it required here that the field not be nil: https://github.com/opentdf/platform/blob/9a898e24673fca8185192102208f299f2e09c0c8/service/policy/attributes/attributes.proto#L172